packages/block-library/src/missing/edit.native.js

```diff
@@ -48,11 +44,38 @@ export class UnsupportedBlockEdit extends Component {
 	}
 
 	toggleSheet() {
+		const { attributes, block, clientId } = this.props;
+		const { originalName } = attributes;
+		const title = this.getTitle();
+		const blockContent = serialize( block ? [ block ] : [] );
+
+		if ( this.canEditUnsupportedBlock() ) {
+			requestUnsupportedBlockFallback(
+				blockContent,
+				clientId,
+				originalName,
+				title
+			);
+			return;
+		}
+
 		this.setState( {
 			showHelp: ! this.state.showHelp,
 		} );
 	}
 
+	canEditUnsupportedBlock() {
+		const {
+			canEnableUnsupportedBlockEditor,
+			isUnsupportedBlockEditorSupported,
+		} = this.props;
+
+		return (
+			! canEnableUnsupportedBlockEditor &&
+			isUnsupportedBlockEditorSupported
+		);
+	}
```

Comment:
Oh, I've realized that `canEnableUnsupportedBlockEditor` will be false in case the user needs to change the Jetpack settings. Hence, we'll only display UBE directly from the block in case there's no need to present the bottom sheet 👍.

Reply:
Yup! I will add the `isEditableInUnsupportedBlockEditor` check though, to not allow blocks that don't support it and send that to the standard flow instead of the "Tap to edit" 👍

Reply:
Addressed in https://github.com/WordPress/gutenberg/pull/62273/commits/6a5d15a3e829faa7d8a4966a418ee15dd3534f90